utils/docker/Dockerfile.centos.7

```diff
@@ -57,6 +113,7 @@ RUN yum -y install deltarpm && \
         ndctl \
         numactl \
         numactl-devel \
+        openmpi3-devel \
```

Comment:
Is this needed?

Reply:
Ah yes I see.  Apparently this was something a customer requested but I was unable to find out who or why so yes, let's make it consistent and add a record of it should it cause problems again.